<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_AddBook</class>
 <widget class="QDialog" name="Window_AddBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>271</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_BookName">
        <property name="text">
         <string>BookName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_BookName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_AuthorName">
        <property name="text">
         <string>Author</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_AuthorName"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_Subject">
        <property name="text">
         <string>Subject</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Subject"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_AddBook">
      <property name="text">
       <string>AddBook</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_BookId">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
